frontend/test/metabase/containers/SaveQuestionModal.unit.spec.js

```diff
@@ -111,6 +108,39 @@ function fillForm({ name, description }) {
   }
 }
 
+function checkCall(
+  mockFn,
+  { question, name, id, description, collection_id } = {},
+) {
+  const call = mockFn.mock.calls[0];
+  expect(call.length).toBe(1);
```

Comment:
`expect(mockFn).toHaveBeenCalledTimes(1)`

Reply:
That's how it's done now.